src/_P121_HMC5883L.ino

```diff
@@ -0,0 +1,158 @@
+
+#include "_Plugin_Helper.h"
+#ifdef PLUGIN_BUILD_DEV //at the very beginning
```

Comment:
This `#ifdef PLUGIN_BUILD_DEV` should be removed (and the matching #endif at the end)